docs/hpc/03_storage/07_large_number_of_small_files.md

```diff
@@ -65,15 +75,16 @@ Essentially, this is a large file sitting on the disk that contains a lot of sma
 
 This is a memory-mapped database meaning, file can be larger than RAM. OS is responsible for managing the pages (like caching frequently uses pages).
 
-For practical use it means: say you have 10 GB of RAM, and LMDB file of 100 GB. When you connect to this file, OS may deside to load 5GB to RAM, and the rest 95GB will be attached as virtual memory. PRINCE does not have limit for virtual memory. Of course, if your RAM is larger than LMDB file, this database will perform the best, as OS will have enough resources to keep what is needed directly in RAM.
+For practical use it means: say you have 10 GB of RAM, and LMDB file of 100 GB. When you connect to this file, OS may decide to load 5GB to RAM, and the rest 95GB will be attached as virtual memory. PRINCE does not have limit for virtual memory. Of course, if your RAM is larger than LMDB file, this database will perform the best, as OS will have enough resources to keep what is needed directly in RAM.
```

Comment:
Remove all mentions of PRINCE. It was retired before both of us joined!

Reply:
Sounds good.  I had asked Lior if that same statement is true of Greene and Francis, but haven't heard back yet.  I was just going to swap those names out if true and delete if false.  Do you know if that's true for the current clusters?

Similarly with the mention of Prince in the [Benchmarking Code section](https://github.com/NYU-RTS/rts-docs-dev/blob/usability/docs/hpc/03_storage/07_large_number_of_small_files.md#benchmarking-code).  Should we try running the benchmarking code on Greene and then Francis or just remove the section?

Reply:
>Should we try running the benchmarking code on Greene and then Francis or just remove the section?

Ideally yes. I'd say this might be a good candidate for a VIP project in the fall. What do you think?

Reply:
Oh, sure, sounds like a great idea.